frontend/src/components/Table/Table.vue

```diff
@@ -0,0 +1,198 @@
+<template>
+  <div class="container mx-auto">
+    <table class="min-w-full bg-white">
+      <thead>
+        <tr>
+          <th
+            v-for="dessertkey in dessertkeys"
+            :key="dessertkey"
+            class="py-2 px-4 border-b text-left"
+          >
+            <div class="flex">
+              <span>{{ dessertkey.toUpperCase() }}</span>
+              <Icon
+                icon="mdi:arrow-down"
+                width="24"
+                color="#808080"
+                @click="sortDessert(dessertkey)"
+              />
```

Comment:
suggestion: the cursor can change to a pointer when the arrow down icon is hovered with the mouse, so that it's more evident it is clickable